requirements/test.txt

```diff
@@ -15,7 +15,6 @@ phmdoctest>=1.1.1
 cloudpickle>=1.3
 scikit-learn>=0.24
 scikit-image>0.17.1
-nltk>=3.6
```

Comment:
Neat !

Reply:
why? it is just moved as direct dependency